docs/user-guide/api/classifiers.rst

```diff
@@ -14,6 +14,12 @@ Classifiers
 .. autoclass:: nemo_curator.classifiers.FineWebEduClassifier
     :members:
 
+.. autoclass:: nemo_curator.classifiers.FineWebMixtralClassifier
```

Comment:
```suggestion
.. autoclass:: nemo_curator.classifiers.FineWebEduMixtralClassifier
```

Reply:
On Hugging Face it will be `FineWebMixtralEduClassifier`, not `FineWebEduMixtralClassifier`. I can update to that instead.